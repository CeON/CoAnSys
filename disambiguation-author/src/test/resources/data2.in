c1	1	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
c2	2	Kowalski	[EX_TITLE#{(An Empirical Study on the AND)},EX_YEAR#{(2011)}]
c3	3	Kowalski	[EX_TITLE#{(An Empirical Study on the BAND)},EX_YEAR#{(2011)}]
c4	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul)},EX_YEAR#{(1996)}]
c5	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 2)},EX_YEAR#{(1996)}]
c6	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 3)},EX_YEAR#{(1996)}]
c7	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Dlugi Tytul 4)},EX_YEAR#{(1996)}]
c8	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c9	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c10	1	Nowak	[EX_TITLE#{(Kosmiczny Tytul)},EX_YEAR#{(1996)}]
c11	1	Nowak	[EX_TITLE#{(Kosmiczny Calkiem Tytul 2)},EX_YEAR#{(1996)}]
c12	1	Nowak	[EX_TITLE#{(Samotnik)},EX_YEAR#{(2001)}]
