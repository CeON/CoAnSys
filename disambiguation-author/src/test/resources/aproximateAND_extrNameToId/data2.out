({(d8,Wos,[7#{(2000)},A#{(Tit)}]),(d7,Wos,[7#{(2000)},A#{(Tit)}]),(d15,Wos,[7#{(2000)},A#{(Tit)}]),(d14,Wos,[7#{(2000)},A#{(Tit)}]),(d13,Wos,[7#{(2000)},A#{(Tit)}]),(d1,Wos,[7#{(2000)},A#{(Tit)}]),(d2,Wos,[7#{(2000)},A#{(Tit)}]),(d3,Wos,[7#{(2000)},A#{(Tit)}]),(d4,Wos,[7#{(2000)},A#{(Tit)}]),(d5,Wos,[7#{(2000)},A#{(Tit)}]),(d6,Wos,[7#{(2000)},A#{(Tit)}]),(d12,Wos,[7#{(2000)},A#{(Tit)}]),(d11,Wos,[7#{(2000)},A#{(Tit)}]),(d10,Wos,[7#{(2000)},A#{(Tit)}]),(d9,Wos,[7#{(2000)},A#{(Tit)}])},{(1,0,1.0),(2,0,1.0),(3,0,1.0),(4,0,1.0),(5,0,1.0),(6,0,1.0),(7,0,1.0),(8,0,1.0),(9,0,1.0),(10,0,1.0),(11,0,1.0),(12,0,1.0),(13,0,1.0),(14,0,1.0)})
({(c9,Nowak,[7#{(1990)},A#{(An Empirical Study on the BRAND)}]),(c1,Nowak,[7#{(2011)},A#{(An Empirical Study on the AND)}]),(c2,Nowak,[7#{(2012)},A#{(An Empirical Study on the AND)}]),(c3,Nowak,[7#{(2013)},A#{(An Empirical Study on the BAND)}]),(c4,Nowak,[7#{(2014)},A#{(An Empirical Study on the BAND)}]),(c5,Nowak,[7#{(2014)},A#{(An Empirical Study on the AND)}]),(c6,Nowak,[7#{(1980)},A#{(An Empirical Study on the TRAND)}]),(c7,Nowak,[7#{(1980)},A#{(An Empirical Study on the BRAND)}]),(c8,Nowak,[7#{(1990)},A#{(An Empirical Study on the SAND)}]),(c10,Nowak,[7#{(2000)},A#{(An Empirical Study on the AND)}]),(c11,Nowak,[7#{(2000)},A#{(An Empirical Study on the TRAND)}])},{(2,1,0.0),(4,3,0.0),(5,1,0.0),(5,4,0.0),(7,0,0.0),(7,6,0.0),(8,0,0.0),(9,1,0.0),(10,6,0.0),(10,9,0.0)})
({(b4,Kowalski,[7#{(1990)},A#{(An Empirical Study on the BRAND)}]),(b3,Kowalski,[7#{(1990)},A#{(An Empirical Study on the SAND)}]),(b2,Kowalski,[7#{(1980)},A#{(An Empirical Study on the BRAND)}]),(b1,Kowalski,[7#{(1980)},A#{(An Empirical Study on the TRAND)}])},{(1,0,0.0),(2,0,0.0),(3,2,0.0)})
({(a5,Kowalski,[7#{(2014)},A#{(An Empirical Study on the AND)}]),(a4,Kowalski,[7#{(2014)},A#{(An Empirical Study on the BAND)}]),(a3,Kowalski,[7#{(2013)},A#{(An Empirical Study on the BAND)}]),(a2,Kowalski,[7#{(2012)},A#{(An Empirical Study on the AND)}]),(a1,Kowalski,[7#{(2011)},A#{(An Empirical Study on the AND)}])},{(1,0,0.0),(2,1,0.0),(3,0,0.0),(4,0,0.0)})
